setup_flashgg.sh

```diff
@@ -79,6 +79,11 @@ cp XGBoostCMSSW/XGBoostInterface/toolbox/*xml $CMSSW_BASE/config/toolbox/$SCRAM_
 scram setup rabit
 scram setup xgboost
 
+# Grab xml file for top-tagger BDT from ttH MultiLepton analysis (too large to store in Github)
+pushd flashgg/Taggers/data/
+wget "http://uaf-8.t2.ucsd.edu/~sjmay/ttH/resTop_xgb_csv_order_deepCTag.xml"
```

Comment:
I am considering moving all the weights files to EOS keeping only a sym-link in the repo. Can you easily access EOS from your T2?

Reply:
Yes, that would be a better solution than what I've done. I believe I can access /eos easily from the T2 (I'll let you know if we're unable to)